### PR TITLE
Shareable components

### PR DESCRIPTION
**Pull request by styled-man**

### CODE REVIEW

**Review by raidel-a — APPROVED** [review]
Lgtm
Can we add the tailwind autosort class plugin??

**Comment by styled-man** [comment]
> Lgtm Can we add the tailwind autosort class plugin??

That might get annoying for the people that use autosave

File: src/components/Button.tsx [hunk 1]
```diff
@@ -1,20 +1,63 @@
...
   return (
-    <>
-      <a
-        className="inline-flex w-full items-center justify-center gap-2 rounded-md bg-primary-yellow px-3 py-2 text-xs font-medium text-gray-700 shadow-sm transition-all hover:bg-light_yellow focus:outline-none focus:ring-2 focus:ring-light_yellow sm:rounded-lg sm:px-4 sm:py-3 sm:text-sm"
-        href={buttonLink}
-        target="_blank">
-        {children}
-      </a>
-    </>
+    <GradientBorder
```
Comment (raidel-a) [inline src/components/Button.tsx:52]: seeing a small visual bug on the button with gradientBorder where onclick the animated gradient translates -1 but the black mask does not.

**Review by raidel-a — CHANGES_REQUESTED** [review]
animated gradient has visual bug onclick

File: src/components/Button.tsx
Comment (02danieljonas) [inline src/components/Button.tsx:1]: Modify the code so that the animated button can be set to always active, instead of active on hover.

Comment (02danieljonas) [inline src/components/Button.tsx:1]: Modify Link code so that a target can be set, i.e. target="_blank"

**Review by 02danieljonas — CHANGES_REQUESTED** [review]

File: src/components/Button.tsx
Comment (02danieljonas) [reply src/components/Button.tsx:1]: In cases where it is always active you can set the glow to be on hover and the border to be animated always

Comment (02danieljonas) [reply src/components/Button.tsx:1]: Passing a classname of rounded-xl modifies the outside element but leaves the inner element alone.

**Review by 02danieljonas — CHANGES_REQUESTED** [review]

**Review by raidel-a — APPROVED** [review]
LGTM

**Review by 02danieljonas — APPROVED** [review]
LGTM!!

**Review by raidel-a — APPROVED** [review]
LGT Daniel

**Pull request merged**